gravitee-am-repository/gravitee-am-repository-mongodb/src/main/java/io/gravitee/am/repository/mongodb/management/MongoI18nDictionaryRepository.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.repository.mongodb.management;
+
+import com.mongodb.reactivestreams.client.MongoCollection;
+import io.gravitee.am.model.I18nDictionary;
+import io.gravitee.am.model.ReferenceType;
+import io.gravitee.am.repository.management.api.I18nDictionaryRepository;
+import io.gravitee.am.repository.mongodb.management.internal.model.I18nDictionaryMongo;
+import io.reactivex.Completable;
+import io.reactivex.Flowable;
+import io.reactivex.Maybe;
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import org.bson.Document;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+import java.util.Objects;
+
+import static com.mongodb.client.model.Filters.and;
+import static com.mongodb.client.model.Filters.eq;
+import static io.gravitee.am.common.utils.RandomString.generate;
+import static io.gravitee.am.model.ReferenceType.valueOf;
+
+@Component
+public class MongoI18nDictionaryRepository extends AbstractManagementMongoRepository implements I18nDictionaryRepository {
+
+    private MongoCollection<I18nDictionaryMongo> mongoCollection;
+
+    @PostConstruct
+    public void init() {
+        mongoCollection = mongoOperations.getCollection("dictionaries", I18nDictionaryMongo.class);
```

Comment:
```suggestion
        mongoCollection = mongoOperations.getCollection("i18n_dictionaries", I18nDictionaryMongo.class);
```
As we name the object `I18nDictionary` at REST API level because we may have in the future other type of dictionaries, may be we could also name the collection `i18n_dictionaries` for consistency